core/src/commitment.rs

```diff
@@ -326,6 +357,42 @@ mod tests {
         );
     }
 
+    #[test]
+    fn test_get_confirmations() {
```

Comment:
Should also have a test to make sure that the highest number of confirmations is returned

Reply:
It's hard to see in the unit tests, since we're hand-rolling the stake and can do whatever we want, but the way the aggregation works, there won't ever be more than one confirmation_count that has >50% of total stake.